### PR TITLE
Kana: Updating to the text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/kana

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @hradecek

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/Kana.t [hunk 1]
```diff
@@ -9,162 +9,57 @@ use utf8;
 zci answer_type => 'kana';
 zci is_cached => 1;
 
+sub build_test
+{
```
Comment (GuiltyDolphin) [inline t/Kana.t]: @mintsoft Cwuddle!

Comment (mintsoft) [inline t/Kana.t]: Old habits!

Comment (GuiltyDolphin) [inline t/Kana.t]: Bad habits - for Perl at least :stuck_out_tongue:

Comment (GuiltyDolphin) [inline t/Kana.t]: @mintsoft Have your editor chastise you every time you do it!

Comment (mintsoft) [inline t/Kana.t]: I would but I never use the same editor twice!

Comment (GuiltyDolphin) [inline t/Kana.t]: @mintsoft I think you need to settle :wink: - the downside is it becomes extremely painful to use anything else (other than Emacs or Vim for me).

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks! :dolphin:

**Pull request merged**